src/ui/components/search/searchcomponent.js

```diff
@@ -280,6 +284,32 @@ export default class SearchComponent extends Component {
     if (!this._isTwin) {
       this.initQueryUpdateListener();
     }
+
+    /**
+     * If true, a loading indicator will be displayed
+     * @type {boolean}
+     */
+    this._showLoadingIndicator = config.loadingIndicator?.display || false;
+
+    /**
+     * Custom icon url for loading indicator
+     * @type {string}
+     */
+    this.loadingIconUrl = config.loadingIndicator?.iconUrl || null;
```

Comment:
I konw I just suggested this name, but now I'm thinking renaming this to `customLoadingIconUrl` might be clearer

Reply:
yup. make sense, like the other variable name`customIconUrl`..should've thought of that before